test/controllers/RemoveNiphlControllerSpec.scala

```diff
@@ -82,8 +84,6 @@ class RemoveNiphlControllerSpec extends SpecBase with MockitoSugar {
 
       when(mockSessionRepository.set(any())) thenReturn Future.successful(true)
 
-      val mockTraderProfileConnector = mock[TraderProfileConnector]
```

Comment:
is mockTraderProfileConnector being reset anywhere? 
Assuming this had its own one to avoid any other test affecting the verify on line 106?

Reply:
No. I removed this because it was a duplicate, already declared on line 50. Do you think this needs to be revert?

Reply:
If it works I'm happy. Was just trying to figure out the reason it was declared separately.